<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shedule</class>
 <widget class="QDialog" name="shedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>1</y>
     <width>320</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="choose18">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>85</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>18:00</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_pic">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>461</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Выбрать другой день</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="choose10">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>25</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>10:00</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>800</width>
     <height>430</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>450</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page_5"/>
   <widget class="QWidget" name="page_6">
    <widget class="QPushButton" name="back_5">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>260</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>назад</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit10_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit10_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="signup10am">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>260</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Записаться</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Введите ваш номер телефона</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Введите ваше ФИО</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_7">
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Введите ваш номер телефона</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signup14pm">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>260</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Записаться</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Введите ваше ФИО</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit14_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="back_6">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>260</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>назад</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit14_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_8">
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>150</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Введите ваш номер телефона</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signup18pm">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>260</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Записаться</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Введите ваше ФИО</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit18_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="back_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>260</y>
       <width>101</width>
       <height>32</height>
      </rect>
     </property>
     <property name="baseSize">
      <size>
       <width>10</width>
       <height>10</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>назад</string>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit18_4">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>471</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <zorder>label_13</zorder>
    <zorder>label_14</zorder>
    <zorder>lineEdit18_3</zorder>
    <zorder>back_2</zorder>
    <zorder>lineEdit18_4</zorder>
    <zorder>signup18pm</zorder>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>221</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="choose14">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>55</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>10</width>
     <height>10</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>14:00</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_pic</zorder>
  <zorder>exit</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>tableWidget</zorder>
  <zorder>choose10</zorder>
  <zorder>choose14</zorder>
  <zorder>choose18</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
